src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -68,27 +87,46 @@ export class TemplateEditorComponent implements OnInit {
    */
   ngOnInit(): void {
     if (this.templateObject.docType && this.templateObject.templateKey) {
+      this.templateEditor.controls.templateKey.setValue(this.templateObject.templateKey);
```

Comment:
These 3 lines could be replaced with:
`this.templateEditor.patchValue(this.templateObject)`

Reply:
Changed to patchValue.